README.md

```diff
@@ -50,6 +50,28 @@ $ npx ember-angle-brackets-codemod angle-brackets app/templates
 <Ui::Button @text="Click me" />
 ```
 
+## Advanced Usage
+
+To help the codemod disambiguate components and helpers, you can define a list of helpers from your application in a configuration file as follows:
```

Comment:
```suggestion
To help the codemod disambiguate components and helpers, you can define (or blacklist) a list of helpers from your application in a configuration file as follows:
```

Reply:
I'm not sure about this suggested change. You currently can just define a single list of helpers so I'm not sure we need to mention the `(or blacklist)`

Reply:
@GavinJoyce , As @jacobq mentioned it in #12 , I personally liked the term `blacklist` since it conveys the purpose in a more direct way, without the need to think what this list of helpers are for in the config, that's why i suggested the change

I am fine with that, if it is not necessary to mention it in the readme